internal/data_generator/internal/.nocover

```diff
@@ -0,0 +1 @@
+FIXME: We test the generated code.
```

Comment:
What does this FIXME do?

Reply:
This script https://github.com/open-telemetry/opentelemetry-collector/blob/master/scripts/check-test-files.sh looks for all packages to have unittests but I don't see a reason to have them for this, and the .nocover does that.

If you ask just for the `FIXME` word that should be removed.

Reply:
Fixed in #688